### ISSUE TITLE
[Tech debt] Clean up `atd_txdot_locations` table

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#16779]
The `atd_txdot_locations` table appears to have a lot of columns that are not needed for VZE, VZV, ETL, or other VZ reporting.

For the purposes of the data model work, the task at hand is to identify columns which can potentially be removed. Once this  task is complete, John will work with Xavier to finalize the changes that we're targeting for this table.

**Comment by roseeichelmann** [comment]
@johnclary 
Filled in the [spreadsheet here](https://docs.google.com/spreadsheets/d/1oHzKt8Cvd1BGpZ7KJl4dfBDHGuSl7gs4yN0OM7AJMBA/edit#gid=0) for `atd_txdot_locations`

The only columns i think we need to keep are:

location_id
description
last_update
latitude
longitude
location_group
scale_factor
shape
street_level

**Comment by johnclary** [comment]
thanks @roseeichelmann! going to move this to review/QA and leave it there for now

**Comment by johnclary** [comment]
This work has been merged into the layer data model. I tagged this issue accordingly. It's done!
